src/Home.jsx

```diff
@@ -1,9 +1,108 @@
 import React from 'react';
+import axios from 'axios';
 
-function Home(props) {
-  return (
-    <h1>Skill Directory Home</h1>
-  );
+const api = (process.env.REACT_APP_API);
+
+class Home extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { 
+      totalTeamMembers: null, 
+      totalSkills: null,
+      recentSkillReviews: null,
+     };
+  }
+
+  componentDidMount() {
+    getTotalTeamMembers(api, total => {
+      this.setState({ totalTeamMembers: total })
+    });
+    getTotalSkills(api, total => {
+      this.setState({ totalSkills: total })
+    });
+    getRecentSkillReviews(api, reviews => {
+      this.setState({ recentSkillReviews: reviews });
+    }, 5);
+  }
+
+  render() {
+    let skillReviews = null;
+    if(this.state.recentSkillReviews != null) {
+      skillReviews = this.state.recentSkillReviews.map(skillReview =>
+        <li key={skillReview.id}>
+            <div>
+              {skillReview.team_member_name 
+               + ' reviewed the ' + 
+               skillReview.skill_name 
+               + ' skill:'}
```

Comment:
Nitpick, but why not use [ES6 template string syntax?](https://github.com/airbnb/javascript#es6-template-literals)